include/LeMonADE/updater/moves/MoveAddBccMonomer.h

```diff
@@ -41,9 +41,11 @@ along with LeMonADE.  If not, see <http://www.gnu.org/licenses/>.
  *
  * @todo Code doubling with MoveAddScMonomer! Specialized class only neccessary to enable the features to handle the moves types differently.
  **/
-class MoveAddBccMonomer:public MoveBase
+class MoveAddBccMonomer:public MoveAddMonomerBase<MoveAddBccMonomer>
```

Comment:
We should think about a standard naming convention. As for MoveLocalBase and its derived classes MoveLocalBcc and MoveLocalSc the parent class is named MoveAddMonomerBase and this class should named MoveAddMonomerBcc instead of MoveAddBccMonomer.